docker/Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM ubuntu:20.04
+FROM golang:alpine as gobuild
 
-# Make sure that the underlying container is patched to the latest versions
-RUN apt-get update && \
-    apt-get install -y gzip tar wget
+RUN apk update && \
+    apk add ca-certificates gcc git make musl-dev && \
+    git clone https://github.com/mattermost/focalboard && \
+    cd focalboard && \
+    make server-linux
 
-# Now install Focalboard as a seperate layer
-RUN wget https://github.com/mattermost/focalboard/releases/download/v0.6.1/focalboard-server-linux-amd64.tar.gz && \
-    tar -xvzf focalboard-server-linux-amd64.tar.gz && \
-    mv focalboard /opt
+FROM node:alpine as nodebuild
 
-EXPOSE 8000
+RUN apk update && \
+    apk add ca-certificates git && \
+    git clone https://github.com/mattermost/focalboard && \
```

Comment:
Why have a second clone here? why not just copy the files over from the prior stage?